onnxruntime/test/onnx/main.cc

```diff
@@ -480,6 +480,48 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"bitshift_right_uint16", "BitShift(11) uint16 support not enabled currently"},
       {"bitshift_left_uint16", "BitShift(11) uint16 support not enabled currently"},
       {"maxunpool_export_with_output_shape", "Invalid output in ONNX test. See https://github.com/onnx/onnx/issues/2398"},
+      {"averagepool_2d_ceil", "shape mismatch", {"onnx150", "onnxtip"}},
+      {"batchnorm_epsilon_training_mode", "Training_mode is not a scalar boolean", {"onnxtip"}},
+      {"batchnorm_example_training_mode", "Training_mode is not a scalar boolean", {"onnxtip"}},
+      {"cast_FLOAT_to_STRING", "result differs", {"onnx141", "onnx150", "onnxtip"}},
+      {"cast_STRING_to_FLOAT", "the pre-allocate size does not match the size in proto", {"onnx141"}},
+      {"convtranspose_3d", "Non-zero status code returned while running ConvTranspose node", {"onnx130", "onnx141", "onnx150", "onnxtip"}},
+      {"cumsum_1d_reverse_exclusive", "result differs", {"onnxtip"}},
+      {"dropout_default", "result differs", {"onnxtip"}},
```

Comment:
Why dropout starts to fail? If it gets changed in the new ONNX version, we shouldn't have an implementation for it. Why it is failing?

Reply:
it’s failing because reference implementation in ONNX repo is incorrect and produces incorrect baseline. This reference implementation was changed in february,